src/curated-corpus/helpers/helperFunctions.ts

```diff
@@ -144,3 +145,24 @@ export const getLocalDateTimeForGuid = (
     .toFormat('DDD')
     .concat(', ', localDateTime.toFormat('t'));
 };
+
+/**
+ * Converts an user's Mozilla ldap username e.g ad|Mozilla-LDAP|jdoe to jdoe
+ * @param ldapString
+ * @returns Curator's username in the following format: flastname
+ */
+export const getCuratorNameFromLdap = (ldapString: string): string => {
+  return ldapString.split('|')[2] ?? 'invalid ldap string';
```

Comment:
maybe the default case here should just be `ldapString`? that way the actual (weird) value will at least be visible - both for curators to see and us to debug 😄

Reply:
Sure, I was just wondering from a security point of view if we wanted to expose someone's LDAP string but since it's an internal tool, it should be fine.